proto/kava/auction/v1beta1/genesis.proto

```diff
@@ -25,14 +25,15 @@ message Params {
 
   google.protobuf.Duration max_auction_duration = 1 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
 
-  google.protobuf.Duration bid_duration = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+  google.protobuf.Duration forward_bid_duration = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+  google.protobuf.Duration reverse_bid_duration = 3 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
 
-  bytes increment_surplus = 3
+  bytes increment_surplus = 4
       [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
 
-  bytes increment_debt = 4
+  bytes increment_debt = 5
       [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
 
-  bytes increment_collateral = 5
+  bytes increment_collateral = 6
```

Comment:
It's best to not change field numbers in proto files. The don't need to be sequential, they're just unique ids.

Don't think it actually matters in this case of json genesis files. But generally if you keep field numbers the same it allows data encoded with an old proto file to be read by code with an updated proto file. Or the otherway round; some old code can still read messages sent to it encoded with a new proto file.

In this case you could do something like leave `bid_duration` as it and use as the forward bid duration, and add a new field `reverse_bid_duration` with id 6.

Reply:
Right, thought breaking changes would be fine but makes more sense if the proto files are backwards compatible until next versions